### PR TITLE
Fixed carousel not showing under iPad view

### LINKED ISSUE #756: 🐛 UI - Image viewer not compatible in iPad view
cc @bradystroud @adamcogan @tiagov8 

**Describe the bug**
The image viewer on this page https://www.ssw.com.au/offices/newcastle is not compatible and does not show up when under iPad view

**To Reproduce**
1. Go to https://www.ssw.com.au/offices/newcastle
2. Switch to iPad view
3. Image viewer not showing up anymore

**Expected behavior**
Image viewer should be compatible under any device 

**Screenshots/Video**
image

Figure: Image viewer working fine on Desktop view

image

Figure: Image viewer not showing anymore when switch to iPad view

1. Please fix to make image viewer compatible to any device

### PR DESCRIPTION
**Pull request by tombui99**
Fixes https://github.com/SSWConsulting/SSW.Website/issues/756

Affected routes:

- https://www.ssw.com.au/offices/newcastle

Add done video, screenshots

image

**Figure: Toggle show/hide carousel on mobile device**

### CODE REVIEW

**Review by Harry-Ross — CHANGES_REQUESTED** [review]
It would be better to add a toggle boolean to the `carouselBlockSchema` for if we want the carousel to be displayed on mobile devices. At the moment, the `md:flex` was chosen to hide the carousel when on mobile devices on the homepage like so:
image
**Figure: iPad display w/ hidden carousel**

It would be much better to have a tickbox to decide whether to display the carousel, to ensure that all mobile devices (iPad and phones) can see the carousel on other pages, such as `/offices/newcastle`.

**Review by amankumarrr — APPROVED** [review]
LGTM

**Review by amankumarrr — APPROVED** [review]

File: components/blocks/carousel.tsx [hunk 1]
```diff
@@ -28,7 +28,10 @@ export const Carousel = ({ data }) => {
   };
 
   return (
-    <Section className="hidden md:flex" color={data.backgroundColor}>
+    <Section
+      className={`${data.showOnMobileDevices ? "" : "hidden"} md:flex`}
```
Comment (Harry-Ross) [inline components/blocks/carousel.tsx]: Don’t we still want to display: flex on devices smaller than md?

Comment (tombui99) [reply components/blocks/carousel.tsx]: Yes - it works on mobile device as well (tested from iPhone X view)

**Review by Harry-Ross — APPROVED** [review]
lgtm

**Pull request merged**